demos/instr.h

```diff
@@ -24,3 +30,60 @@ void CLFlush(const void *memory);
 
 // Measures the latency of memory read from a given address
 uint64_t ReadLatency(const void *memory);
+
+#ifdef __GNUC__
+// Inlines the afterspeculation label.
+__attribute__((always_inline))
+inline void afterspeculate() {
```

Comment:
I don't think it makes sense to hide this in a function. For example, this function can't be called twice without causing some obscure linker errors.

In the limit I'd like to remove the necessity for the `afterspeculation` label entirely, but in the meantime let's leave this embedded in the code actually defining the label.

Reply:
Done. I was just following your instruction to move all inline assembly outside.